tasktiger/worker.py

```diff
@@ -11,16 +11,20 @@
 import threading
 import time
 import traceback
+import uuid
 
 from .redis_lock import Lock
 
 from ._internal import *
 from .exceptions import RetryException, TaskNotFound
+from .redis_semaphore import Semaphore
 from .retry import *
 from .stats import StatsThread
 from .task import Task
 from .timeouts import UnixSignalDeathPenalty, JobTimeoutException
 
+LOCK_REDIS_KEY = 'qlock'
```

Comment:
Should consider adding this to the "Redis keys" comment section in `tasktiger/__init__.py`.